lib/streamlit/runtime/caching/legacy_cache_api.py

```diff
@@ -36,6 +36,7 @@ def cache(
     persist: bool = False,
     allow_output_mutation: bool = False,
     show_spinner: bool = True,
+    show_time: bool = False,
```

Comment:
I don't think we want to add any new parameters tl `st.cache` since it is deprecated and will be removed soon. It's fine to just have this for `st.cache_data` and `st.cache_resource`.